Description: Fix spelling mistakes found by lintian
Author: Peter Colberg <peter@colberg.org>
Forwarded: https://github.com/LubosD/twinkle/pull/86
Last-Update: 2016-10-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gui/lang/twinkle_cs.ts
+++ b/src/gui/lang/twinkle_cs.ts
@@ -5913,7 +5913,7 @@
         <translation>&amp;Kvalita:</translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation>Speex je ztrátový kodek, což znamená, že je na úkor kvality možné docílit redukce datového toku. Na rozdíl od jiných hlasových kodeků je možné nastavit kompromis mezi kvalitou a datovým tokem. Kódovací proces u tohoto kodeku je zpravidla řízen nastavením parametru kvality v rozsahu od 0 do 10.</translation>
     </message>
     <message>
diff --git a/src/gui/lang/twinkle_de.ts b/src/gui/lang/twinkle_de.ts
index 99a9c94..869221f 100644
--- a/src/gui/lang/twinkle_de.ts
+++ b/src/gui/lang/twinkle_de.ts
@@ -5969,7 +5969,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/gui/lang/twinkle_fr.ts b/src/gui/lang/twinkle_fr.ts
index f32bc65..71f394c 100644
--- a/src/gui/lang/twinkle_fr.ts
+++ b/src/gui/lang/twinkle_fr.ts
@@ -5959,7 +5959,7 @@
         <translation>&amp;Qualité:</translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation>Speex est un codec à perte, ce qui signifie qu&apos;il assure la compression au dépend de la fidélité au son d&apos;entrée. A la différence de quelques autres codecs, il est possible de contrôler l&apos;équilibre entre qualité et compression. Le pocessus d&apos;encodage Speex est contrôlé la plupart du temps par un paramêtre de qualité entre 0 et 10.</translation>
     </message>
     <message>
diff --git a/src/gui/lang/twinkle_nl.ts b/src/gui/lang/twinkle_nl.ts
index 6e289d3..131094e 100644
--- a/src/gui/lang/twinkle_nl.ts
+++ b/src/gui/lang/twinkle_nl.ts
@@ -6017,7 +6017,7 @@
         <translation>&amp;Kwaliteit:</translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation>Speex comprimeert het geluidssignaal ten koste van de kwaliteit. Hoe meer compressie, hoe minder bandbreedte nodig is, maar hoe slechter de geluidskwaliteit. Deze kwaliteitsfactor (0 tot 10) bepaalt de trade-off tussen kwaliteit en compressie.</translation>
     </message>
     <message>
diff --git a/src/gui/lang/twinkle_ru.ts b/src/gui/lang/twinkle_ru.ts
index e6e1fd6..5e7e07f 100644
--- a/src/gui/lang/twinkle_ru.ts
+++ b/src/gui/lang/twinkle_ru.ts
@@ -5407,7 +5407,7 @@
         <translation>&amp;Качество:</translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff --git a/src/gui/lang/twinkle_sv.ts b/src/gui/lang/twinkle_sv.ts
index 31a2dc9..d8b0df6 100644
--- a/src/gui/lang/twinkle_sv.ts
+++ b/src/gui/lang/twinkle_sv.ts
@@ -5659,7 +5659,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/gui/lang/twinkle_xx.ts b/src/gui/lang/twinkle_xx.ts
index d47111b..067913e 100644
--- a/src/gui/lang/twinkle_xx.ts
+++ b/src/gui/lang/twinkle_xx.ts
@@ -5382,7 +5382,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
+        <source>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/gui/userprofileform.ui b/src/gui/userprofileform.ui
index b41508b..61fe2da 100644
--- a/src/gui/userprofileform.ui
+++ b/src/gui/userprofileform.ui
@@ -1498,7 +1498,7 @@
                 <item row="0" column="2">
                  <widget class="QSpinBox" name="spxQualitySpinBox">
                   <property name="whatsThis">
-                   <string>Speex is a lossy codec, which means that it achives compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</string>
+                   <string>Speex is a lossy codec, which means that it achieves compression at the expense of fidelity of the input speech signal. Unlike some other speech codecs, it is possible to control the tradeoff made between quality and bit-rate. The Speex encoding process is controlled most of the time by a quality parameter that ranges from 0 to 10.</string>
                   </property>
                   <property name="minimum">
                    <number>0</number>
diff --git a/src/parser/hdr_warning.h b/src/parser/hdr_warning.h
index 124f3fb..8b7550b 100644
--- a/src/parser/hdr_warning.h
+++ b/src/parser/hdr_warning.h
@@ -50,7 +50,7 @@
 #define WARNING_330	"Multicast not available"
 #define WARNING_331	"Unicast not available"
 #define WARNING_370	"Insufficient bandwidth"
-#define WARNING_399	"Miscellanous warning"
+#define WARNING_399	"Miscellaneous warning"
 
 using namespace std;
 
